terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf

```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
       "arn:aws:s3:::mojap-derived-tables",
       "arn:aws:s3:::dbt-query-dump/*",
       "arn:aws:s3:::dbt-query-dump",
+      "arn:aws:s3:::mojap-manage-offences/*",
+      "arn:aws:s3:::mojap-manage-offences",
+      "arn:aws:s3:::mojap-hub-exports/*",
+      "arn:aws:s3:::mojap-hub-exports/",
```

Comment:
From looking through the code - `bold_sm_spells` is calling `unload` as a macro during their model code - those buckets will be needed too.

Reply:
Actually scratch that, the project is paused.